src/Solver/time_stepping/DirectGhostTimeCluster.h

```diff
@@ -0,0 +1,25 @@
+#ifndef SEISSOL_GHOSTTIMECLUSTER_H
+#define SEISSOL_GHOSTTIMECLUSTER_H
```

Comment:
Headerguard should have same name as file

Reply:
I replaced to `#pragma once`. If you and @krenzland make a decision to keep using header guards then I will revert and rename it